### DISCUSSION TITLE
Support for the Intel C/C++ Compiler

### DISCUSSION THREAD

**Discussion by paspro** [microsoft/vcpkg#17861] [Ideas]
****
I would like to request the addition of the option to use the Intel C/C++ compiler on Windows and Linux as the compiler to use.

**Is your feature request related to a problem? Please describe.**
Many libraries / applications have higher performance when compiled by this compiler.

**Proposed solution**
I suppose that this should be implemented as additional triplets that use the Intel C/C++ compiler instead of MSVC or GCC on Windows and Linux respectively.

**Describe alternatives you've considered**
Hacking vcpkg which is not a good solution.

**Additional context**
-

**Comment by PhoebeHui** [comment]
Related to https://github.com/microsoft/vcpkg/issues/4487

**Reply by ghost** [reply]
Any progress on this?
